docs/dashboard.md

```diff
@@ -97,3 +97,81 @@ index=_internal source=*<addon_name>* ERROR
 ```
 
 > Note: <addon_name> is being replaced by the actual value during the build time.
+
+<br>
+# Custom components
+
+UCC also supports adding your own components to the dashboard. 
```

Comment:
We also need to mention that the autogenerated dashboard has `<input type="time" token="log_time">` so in case an add-on developer wants their panels to be synchronised with the whole dashboard, they need to use this token.

Reply:
```
          <earliest>$log_time.earliest$</earliest>
          <latest>$log_time.latest$</latest>
```